contracts/VerifierRegistry.sol

```diff
@@ -181,4 +167,82 @@ contract VerifierRegistry is Ownable {
     }
     return keccak256(abi.encodePacked(_base));
   }
+
+  function verifiers(address _verifier) external view returns (
```

Comment:
Instead of `verifiers` should be `all`

Reply:
if we change this to `all` we will change current API and it will have consequences for other contracts and services. For now I think we should stay with it.